lucid1/src/Lucid/Base.hs

```diff
@@ -4,7 +4,7 @@
 {-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE GADTs #-}
-{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE TypeOperators #-}
```

Comment:
[`TypeOperators`](https://downloads.haskell.org/ghc/9.12.1-alpha1/docs/users_guide/exts/type_operators.html#extension-TypeOperators
) exists since 6.8 and is needed for [`-Wtype-equality-requires-operators`](https://downloads.haskell.org/ghc/9.12.1-alpha1/docs/users_guide/using-warnings.html#ghc-flag-Wtype-equality-requires-operators)